syft/serde/__init__.py

```diff
@@ -2,3 +2,5 @@
 from syft.serde.torch_serde import *
 from syft.serde.serde import _simplify
 from syft.serde.serde import _detail
+from syft.serde.serde import _compress
```

Comment:
Should these be imported given they are private?

Reply:
we use them all the time in testing, so yes?

Reply:
also in playing around in jupyter notebooks. We could make them public but they're not something you should be using in the official api. It's just nice for dev.

Reply:
Also, this PR is following the current pattern - fixing it should be a separate issue :)

Reply:
Sounds good